packages/dm-core-plugins/src/job/SimpleJobControlButtons.tsx

```diff
@@ -124,3 +134,17 @@ export const CompletedButton = (props: {
     </div>
   )
 }
+
+export const ErrorButton = (props: { error: ErrorResponse }) => {
+  return (
+    <Button
+      style={{
+        backgroundColor: loadingColor,
+        cursor: 'not-allowed',
+      }}
+      variant='contained_icon'
+    >
+      <Icon data={error_outlined} style={{ color: 'white' }} />
```

Comment:
<Icon /> also has a color prop if you want to use that 🙂 
`<Icon data={error_outlined} color="white" />`